ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/router-bgp.j2

```diff
@@ -954,6 +954,9 @@ router bgp {{ router_bgp.as }}
 {%                 if neighbor.route_map_out is arista.avd.defined %}
          neighbor {{ neighbor.ip_address }} route-map {{ neighbor.route_map_out }} out
 {%                 endif %}
+{%                 if neighbor.peer_group is arista.avd.defined %}
```

Comment:
as we discussed, please remove this one again since it is not supported on EOS. it will move the config to the parent context.

Reply:
also remember to remove the molecule testing this.